Octokit/Models/Request/NewPullRequest.cs

```diff
@@ -41,6 +43,11 @@ public NewPullRequest(string title, string head, string baseRef)
         /// </summary>
         public string Head { get; private set; }
 
+        /// <summary>
+        /// Whether maintainers of the base repository can push to <see cref="Head"/> if it is a branch
+        /// </summary>
+        public bool MaintainerCanModify { get; private set; }
```

Comment:
This should be nullable `bool?` since it's an optional parameter.  That way when the user doesnt set it, we dont send anything in the request, and thus we default to whatever the API default is (which is important in this case since the API default is true, and if we leave this non nullable we would default to sending false)